x/stablestake/keeper/msg_server_unbond.go

```diff
@@ -36,24 +39,24 @@ func (k msgServer) Unbond(goCtx context.Context, msg *types.MsgUnbond) (*types.M
 
 	redemptionAmount := shareCoin.Amount.ToLegacyDec().Mul(redemptionRate).RoundInt()
 
-	amountAfterRedemption := params.TotalValue.Sub(redemptionAmount)
-	maxAllowed := (params.TotalValue.ToLegacyDec().Mul(params.MaxWithdrawRatio)).TruncateInt()
+	amountAfterRedemption := pool.TotalValue.Sub(redemptionAmount)
+	maxAllowed := (pool.TotalValue.ToLegacyDec().Mul(pool.MaxLeverageRatio)).TruncateInt()
```

Comment:
Shouldn't this remain params.MaxWithdrawRatio ?

Reply:
yes, changed